locale/de.json

```diff
@@ -382,6 +383,7 @@
     "Mega X":"Mega X",
     "Mega Y":"Mega Y",
     "Mega":"Mega",
+    "membership":"Mitglied",
```

Comment:
Really "Mitglied" or does it mean "Mitgliedschaft"? How is it used in Poracle? ;)